kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -410,10 +410,13 @@ export default {
       return this.postInYourSpacesChoice && !this.audience;
     },
     audienceTypesDisplay() {
-      return (!this.spaceId && !this.isLinkedKudos)  || (!this.spaceId && !this.readOnlySpace) || (!this.readOnlySpace  && this.postInYourSpacesChoice && !this.audience);
+      return (!this.spaceId && !this.isLinkedKudos) || (!this.spaceId && !this.readOnlySpace && !this.isLinkedKudos) || (!this.readOnlySpace  && this.postInYourSpacesChoice && !this.audience);
     },
     audienceAvatarDisplay() {
       return (this.audience && this.postInYourSpacesChoice) || this.readOnlySpace;
+    },
+    kudosSenderAvatar() {
```

Comment:
```suggestion
    displayAudience() {
```

Reply:
ok, then: `displaySenderAvatar` because `kudosSenderAvatar`, its meaning is: This will retrun the avatar of the kudos sender, which is not the case IMO, it's instead: Whether to display or not the avatar.